src/psql_migration.erl

```diff
@@ -193,7 +193,7 @@ connection_opts(Args, {env, URLName}) ->
         DatabaseUrl -> connection_opts(Args, {url, DatabaseUrl})
     end;
 connection_opts(_Args, {url, DatabaseUrl}) ->
-    case uri_string:parse(DatabaseUrl) of
+    case uri_string:parse(re:replace(DatabaseUrl, "\\r|\\n", "", [global,{return,list}])) of
```

Comment:
can you use `string:trim(DatabaseUrl)` and test that it woks please?

Reply:
Yep, that worked too. I have replaced it (no pun intended) with `string:trim`